src/customTypes.d.ts

```diff
@@ -2,37 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { Subscriber } from "./app/(nextjs_migration)/(authenticated)/user/breaches/breaches";
-
-interface HTMLElement {
```

Comment:
These were to tack on types for our partials when they were generated in controllers, and thus are no longer necessary.